<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>510</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
               width: 100%;
            padding: 10px;
            border: none;
			color: rgb(255, 255, 255);
			background-color: rgb(232, 174, 133);
            border-radius: 5px;
			font: 81 16pt &quot;Tajawal ExtraBold&quot;;
            }
QPushButton:hover {
background-color: rgb(190, 141, 109);

            }
</string>
   </property>
   <property name="text">
    <string>‚úàÔ∏èSend</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>1061</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{   
  			width: 100%;
            padding: 10px 30px 10px 30px;
			margin: 10px 0;
            border: none;
            border-radius: 5px;
            outline: none;
            font-size: 16px;
			color: rgb(255, 255, 255);
			background-color: rgba(232, 174, 133,0.4);
}
QLineEdit:focus {
                border: 2px solid rgb(232, 174, 133);
            }


</string>
   </property>
   <property name="placeholderText">
    <string>Enter the massage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voice_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
               width: 100%;
            padding: 10px;
            border: none;
			color: rgb(255, 255, 255);
			background-color: rgb(232, 174, 133);
            border-radius: 5px;
			font: 81 16pt &quot;Tajawal ExtraBold&quot;;
            }
QPushButton:hover {
background-color: rgb(190, 141, 109);

            }
</string>
   </property>
   <property name="text">
    <string>üé§ Voice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
               width: 100%;
            padding: 10px;
            border: none;
			color: rgb(255, 255, 255);
			background-color: rgb(232, 174, 133);
            border-radius: 5px;
			font: 81 16pt &quot;Tajawal ExtraBold&quot;;
            }
QPushButton:hover {
background-color: rgb(190, 141, 109);

            }
</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="chat_display">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>1181</width>
     <height>441</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1179</width>
      <height>439</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="output_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
            width: 30px;
			height:30px;
            border: none;
           border-radius: 15px;
	background-color: none;
  }
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/back (1).png</normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/back (1).png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: transparent; 
           border-radius: 15px;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/no.png</normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/no.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
